### PR TITLE
features/297-Course-detail-page

### PR DESCRIPTION
**Pull request by Horizon19911**
@minkhant285  Thank You Bro!

### CODE REVIEW

**Comment by Horizon19911** [comment]
@minkhant285

File: src/components/CourseDetail/index.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+export function CxDevxCourseDetail({ navigation,route }: HomeStackNavProps<"CourseDetail">){
+    const {image,title} = route.params;
+    const tabNavigation = useNavigation();
+    const parent = tabNavigation.dangerouslyGetParent();
+    let [isLiked,setLike] = useState<Boolean>(false);
+    useEffect(()=>{
+        parent?.setOptions({tabBarVisible: false});
+    },[1]);
```
Comment (minkhant285) [inline src/components/CourseDetail/index.tsx]: why 1

File: src/components/CourseSection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import React from 'react';
```
Comment (minkhant285) [inline src/components/CourseSection/index.tsx]: change index.tsx to courseSection.tsx don't use index.tsx

File: src/Routes/index.tsx [hunk 3]
```diff
@@ -50,17 +51,17 @@ export const Routes: React.FC<RoutesProps> = ({ }) => {
...
+    //         <Center>
+    //             <ActivityIndicator size="large" />
+    //         </Center>
+    //     );
+    // }
 
     return (
         <NavigationContainer>
-            {!isLogined ? <AuthStack /> : <AppDrawer />}
+            {false ? <AuthStack /> : <AppDrawer />}
```
Comment (minkhant285) [inline src/Routes/index.tsx]: why false

**Review by minkhant285 — CHANGES_REQUESTED** [review]

File: src/components/CourseDetail/index.tsx (hunk 1 above)
Comment (Horizon19911) [reply src/components/CourseDetail/index.tsx]: use to hide the bottom tab

**Pull request merged**